### ISSUE TITLE
Loot chance % not accurate

### ISSUE THREAD

**Issue by SewerCzar** [jkachhad/Stormhalter#589]
Unless I've read things incorrectly the loot pack for daisy reads:

            new LootPackEntry(true, (from, container) => new PearDiamond(1500u),    100, gemsPriceMutator),
            new LootPackEntry(true, (from, container) => new ScorpionRobe(),        100)
            
Which I believe should mean there's a 100% chance for a drop of a PearDiamond and a ScorpionRobe. I've killed her 3 times and only pear diamonds have dropped. Saw a similar issue with Giant after I implemented him. Is only the first from, container item dropping?

**Comment by Chilly021** [comment]
@SewerCzar  is this still an issue?

**Comment by SewerCzar** [comment]
I dont believe so. Let me close it  since Ive seen nothing with 100% not drop
